### PR TITLE
Enhance metallb e2e tests code

### PR DESCRIPTION
**Pull request by SurajGudaji**
Created a variable for namespace, prefixed all variables with metallb_ renamed the e2e test log storing files and removed installation task to avoid duplicacy of resources created by openshift-tests-private.

### CODE REVIEW

**Comment by yussufsh** [comment]
Not sure what is the aim of this PR @SurajGudaji 
We already have metallb operator installation and e2e tests in the upstream. See #53

File: playbooks/roles/ocp-metallb-operator/README.md [hunk 1]
```diff
@@ -13,30 +13,28 @@ Requirements
...
+| metallb_enable_global_secret               | no       | true                                                                                 | Set it to true when the secret needs to be updated                                                                             |
+| ocp_version                                | no       | 4.13                                                                                 | Set the proper OCP version to fetch the appropriate catalog source for MetalLB operator                                        |
+| metallb_catalogsource_image                | no       | "quay.io/openshift-release-dev/ocp-release-nightly:iib-int-index-art-operators-4.13" | Custom catalog source index image for MetalLB Operator. If not defined, default `redhat-operators` catalog source will be used |
+| metallb_catalogsource                      | no       | "qe-app-registry"                                                                    | Catalog source channel                                                                                                         |
```
Comment (yussufsh) [inline playbooks/roles/ocp-metallb-operator/README.md:23]:
```suggestion
| metallb_catalog_source                      | no       | "qe-app-registry"                                                                    | Catalog source channel                                                                                                         |
```

File: examples/all.yaml [hunk 2]
```diff
@@ -337,22 +337,21 @@ metallb_enabled: false
...
-metallb_catalogsource_image: "quay.io/openshift-release-dev/ocp-release-nightly:iib-int-index-art-operators-{{ ocp_version }}"
-metallb_catalogsource: "redhat-operators-stage"
-qe_catalog_source_image: "quay.io/openshift-release-dev/ocp-release-nightly:iib-int-index-art-operators-{{ ocp_version }}"
-qe_catalog_source: "qe-app-registry"
+metallb_catalog_source_image: "quay.io/openshift-release-dev/ocp-release-nightly:iib-int-index-art-operators-{{ ocp_version }}"
+metallb_catalog_source: "qe-app-registry"
 metallb_upgrade_channel: "stable"
 metallb_directory: "/tmp/metallb"
+metallb_namespace: "metallb-system"
 metallb_golang_tarball: "https://go.dev/dl/go1.18.6.linux-ppc64le.tar.gz"
```
Comment (yussufsh) [inline examples/all.yaml:345]: 1.18 is EOL.

File: playbooks/roles/ocp-metallb-operator/README.md [hunk 3]
```diff
@@ -13,30 +13,28 @@ Requirements
...
+| metallb_upgrade_channel                    | no       | stable                                                                               | Operator upgrade channel                                                                                                       |
+| metallb_directory                          | no       | `/tmp/metallb`                                                                       | Working directory for MetalLB Operator                                                                                         |
+| metallb_golang_tarball                     | no       | https://go.dev/dl/go1.18.6.linux-ppc64le.tar.gz                                      | HTTPS URL for golang tarball                                                                                                   |
```
Comment (yussufsh) [inline playbooks/roles/ocp-metallb-operator/README.md:26]: The default is not 1.18.

**Review by yussufsh — CHANGES_REQUESTED** [review]

File: examples/metallb_vars.yml [hunk 4]
```diff
@@ -3,26 +3,23 @@ metallb_enabled: false
...
-metallb_catalogsource_image: "quay.io/openshift-release-dev/ocp-release-nightly:iib-int-index-art-operators-{{ ocp_version }}"
-metallb_catalogsource: "redhat-operators-stage"
-qe_catalog_source_image: "quay.io/openshift-release-dev/ocp-release-nightly:iib-int-index-art-operators-{{ ocp_version }}"
-qe_catalog_source: "qe-app-registry"
+metallb_catalog_source_image: "quay.io/openshift-release-dev/ocp-release-nightly:iib-int-index-art-operators-{{ ocp_version }}"
+metallb_catalog_source: "qe-app-registry"
 metallb_upgrade_channel: "stable"
 metallb_directory: "/tmp/metallb"
 metallb_golang_tarball: "https://go.dev/dl/go1.20.6.linux-ppc64le.tar.gz"
+metallb_namespace: "metallb-system"
```
[the remark this answers is not in the document]
Comment (SurajGudaji) [reply examples/metallb_vars.yml:11]: Added metallb_namespace variable in README file

**Review by yussufsh — APPROVED** [review]
/lgtm

**Pull request merged**